### PR TITLE
Actually add Beta Mode

### PR DESCRIPTION
**Pull request by JeffTheRock**
When moving back to main, I forgot to update the payout functions.

Oopsie.

### CODE REVIEW

**Comment by Lekuruu** [comment]
You removed the .example_env file. Please add it again

**Comment by JeffTheRock** [comment]
Readded.

**Comment by Lekuruu** [comment]
Will review it later, thanks!

File: app/engine/cards.py [hunk 1]
```diff
@@ -376,7 +376,8 @@ def consume(self) -> None:
         beam.play()
 
         self.client.ninja.play_sound('SFX_MG_CJSnow_PowercardReviveStart')
-        self.client.ninja.set_health(self.client.ninja.max_hp)
+        self.client.ninja.set_health(0)
```
Comment (Lekuruu) [inline app/engine/cards.py]: Uh.. why?

File: app/engine/game.py [hunk 2]
```diff
@@ -977,6 +986,50 @@ def display_payout(self) -> None:
...
+
+            for client in self.clients:
+                if client.disconnected:
+                    continue
+
+                # Calculate percentage based on round
+                exp_gained = (self.get_payout_round() * 11) + 1
+
+                if not config.DISABLE_REWARDS:
+                    if exp_gained == 100:
```
Comment (Lekuruu) [inline app/engine/game.py]: Maybe make a greater than check here

**Review by Lekuruu — CHANGES_REQUESTED** [review]

File: app/engine/cards.py (hunk 1 above)
Comment (JeffTheRock) [reply app/engine/cards.py]: Oops, testing code.
Removed

File: app/engine/game.py (hunk 2 above)
Comment (JeffTheRock) [reply app/engine/game.py]: Added.

**Review by Lekuruu — APPROVED** [review]
Looking good!

**Pull request merged**